bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java

```diff
@@ -232,6 +231,26 @@ public String getElement() {
         return superTypes;
     }
 
+    public static String inheritWithOverrides(com.day.cq.wcm.api.Page startingPoint, String propertyName) {
```

Comment:
I'd:
- rename the method `getInheritedValue`
- rename `startingPoint` to `startPage`
- add javadocs